.gitignore

```diff
@@ -72,7 +72,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+env/.env
```

Comment:
Does this still work for you if it is stored in the subdirectory?

Reply:
It does if you explicitly specify the path with the `--env-file` flag :) If that's too cumbersome we can move the files back.

Reply:
Too cumbersome IMO